WcaOnRails/app/models/country.rb

```diff
@@ -20,6 +20,10 @@ def self.real
     @real_countries ||= Country.uncached_real
   end
 
+  def real?
+    !read_attribute(:name).include?('Multiple Countries')
```

Comment:
What about using an array of iso codes ? (like `%w(XA XE XS)`)
Your way definitely works, but I feel like being explicit would be clearer.